manifests/Python/Python/3.8.3.yaml

```
@@ -15,3 +15,4 @@ Installers:
       Switches:
         Silent: /Quiet
         SilentWithProgress: /Passive
+        Custom: PrependPath=1
```

```suggestion
        Custom: PrependPath = 1
```
will it looked better?

---

If you add spaces, python does not get added to the path.

---

got it